robottelo/constants.py

```diff
@@ -48,10 +48,12 @@
 
 OPERATING_SYSTEMS = [
```

Comment:
no-no. Everything is fine. I just thought that module is too basic that should not contain any dependencies. Will update code soon